src/apps/pfu.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
```

Comment:
super nitpick. I put whitespace between pragmas and includes, and separate the blocks so it's 
```cpp
pragma

local includes "..."

library includes <...>

system/STL includes <...>

```

but really that's just my style. The rationale for the ordering (which you've kept to here!) is to spot missing `<STL>` includes in header files, which can be hidden if all the c++ files that include them have already included half the STL before they include the header.